params.yml

```diff
@@ -23,6 +23,9 @@ deploy_zkevm_permissionless_node: false
 # Deploy observability stack.
 deploy_observability: true
 
+# Deploy Blockscout stack.
+deploy_blockscout: false
```

Comment:
Does this flag control both the l1 and l2 block scout deployment? it would be good if it controlled both I think. Alternatively, we could make this like `deploy_l2_blockscout` or something like that.

Reply:
Good point! It's just for L2, so I've renamed param to honor it.
For L1 it's set as additional service to ethereum package, here:
https://github.com/0xPolygon/kurtosis-cdk/blob/8c1ca8b3a8310de1449a941ee37b658d53e9eaaa/params.yml#L173